tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+envlist =
+    py27-test
+    py37-test
+    py27-test-clocale
+    py37-test-clocale
+    py27-test-sphinx13
+    py35-test-sphinx14
+    py35-test-sphinx15
+    py36-test-sphinx16
+    py36-test-sphinx17
+    py37-test-sphinx18
+    py37-test-sphinx20
+    py37-test-sphinxdev
+requires = pip >= 18.0
+           setuptools >= 30.3.0
+
+[testenv]
+changedir = .tmp/{envname}
+deps =
+    sphinx13: sphinx==1.3.6
+    sphinx14: sphinx==1.4.9
+    sphinx15: sphinx==1.5.6
+    sphinx16: sphinx==1.6.7
+    sphinx17: sphinx==1.7.9
+    sphinx18: sphinx==1.8.5
+    sphinx20: sphinx==2.0.1
+    sphinx21: sphinx==2.1.2
```

Comment:
is there a mechanism to add new versions here, or we just need to keep an eye out to new releases and add them manually?

Reply:
The latter - but note that the two first builds don't pin sphinx so we'll automatically be testing with the latest.